php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -374,13 +345,6 @@ class NDB_Menu_Filter extends NDB_Menu
             $this->_setSearchKeyword($key);
 
         }
-
-        // 4. If the user clicked on a sort column, set the order.
-        //    It's likely a GET request even though it modifies the data,
-        //    so just use $_REQUEST.
-        if (isset($_REQUEST['filter']['order'])) {
```

Comment:
This seems to be dead code, the filtering/order is done on the react side now. The types get confused because it's a confusing request, so I just removed it (and the test)